manifests/params.pp

```diff
@@ -225,4 +225,7 @@
       $plugin_prefix = 'smart_proxy_'
     }
   }
+
+  $puppet_use_cache = undef
+  $puppet_cache_location = '/var/lib/smart-proxy/cache'
```

Comment:
Packaging does not bring that directory (at least on my DEB installation), so we might have to think of something in the packaging also. Personally I'd put a directory like ```/var/tmp/foreman-proxy/cache``` in place for that, but I'm unsure if that's really the correct place.